packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -97,13 +97,19 @@ export const AuthModal = ({
 
         const handlePasswordlessLogin = async (email) => {
             try {
-                await authorizePasswordlessLogin.mutateAsync({userid: email})
+                const res = await authorizePasswordlessLogin.mutateAsync({userid: email})
+                if (res.status !== 200) {
+                    const errorData = await res.json()
+                    throw new Error(`${res.status} ${errorData.message}`)
+                }
                 setCurrentView(EMAIL_VIEW)
             } catch (error) {
-                form.setError('global', {
-                    type: 'manual',
-                    message: formatMessage(API_ERROR_MESSAGE)
-                })
+                const message = /error getting user info/i.test(error.message)
```

Comment:
@yunakim714 should we also have this error display the `CREATE_ACCOUNT_FIRST_ERROR_MESSAGE`?
```
{
  "status_code" : "404 NOT_FOUND",
  "message" : "User not found"
}
```

Steps to reproduce: 
- enter a email of a user that doesn't exist like `a@s`

Reply:
Sorry now that I'm trying on your env https://wasatch-mrt-social-login-poc.mrt-storefront-staging.com/ I see it does catch this error while my env that uses `zzrf-001` doesn't

I'm going to guess the API has been updated to now return the `error getting user info` instead of `user not found` for when the user does not exist.